static/app/views/settings/project/projectUserFeedback.tsx

```diff
@@ -1,130 +0,0 @@
-import type {RouteComponentProps} from 'react-router';
-import styled from '@emotion/styled';
-import * as Sentry from '@sentry/react';
-
-import Access from 'sentry/components/acl/access';
-import {Button} from 'sentry/components/button';
-import Form from 'sentry/components/forms/form';
-import JsonForm from 'sentry/components/forms/jsonForm';
-import formGroups from 'sentry/data/forms/userFeedback';
-import {t} from 'sentry/locale';
-import {space} from 'sentry/styles/space';
-import type {Organization} from 'sentry/types/organization';
-import type {Project} from 'sentry/types/project';
-import routeTitleGen from 'sentry/utils/routeTitle';
-import withOrganization from 'sentry/utils/withOrganization';
-import DeprecatedAsyncView from 'sentry/views/deprecatedAsyncView';
-import SettingsPageHeader from 'sentry/views/settings/components/settingsPageHeader';
-import TextBlock from 'sentry/views/settings/components/text/textBlock';
-import PermissionAlert from 'sentry/views/settings/project/permissionAlert';
-
-type RouteParams = {
-  projectId: string;
-};
-type Props = RouteComponentProps<RouteParams, {}> & {
-  organization: Organization;
-  project: Project;
-};
-
-class ProjectUserFeedbackSettings extends DeprecatedAsyncView<Props> {
-  submitTimeout: number | undefined = undefined;
-
-  componentDidMount() {
-    super.componentDidMount();
-
-    window.sentryEmbedCallback = function (embed) {
-      // Mock the embed's submit xhr to always be successful
-      // NOTE: this will not have errors if the form is empty
-      embed.submit = function (_body) {
-        this._submitInProgress = true;
-        window.setTimeout(() => {
-          this._submitInProgress = false;
-          this.onSuccess();
-        }, 500);
-      };
-    };
-  }
-
-  componentWillUnmount() {
-    window.sentryEmbedCallback = null;
-  }
```

Comment:
not entirely sure whether this chunk was needed but it seemed to work fine without it

Reply:
cc @billyvg looks like you wrote this code ... 6 years ago😭

Reply:
omg. triple OG code

Reply:
🙈

Reply:
I think we need `window.SentryEmbedCallback` because otherwise the crash modal report will send to sentry